selenium/che-selenium-test/src/test/java/org/eclipse/che/selenium/git/CommitFilesTest.java

```diff
@@ -88,6 +90,7 @@ public void prepare() throws Exception {
     testProjectServiceClient.importProject(
         ws.getId(), Paths.get(resource.toURI()), PROJECT_NAME, ProjectTemplates.MAVEN_SPRING);
     ide.open(ws);
+    consoles.waitJDTLSProjectResolveFinishedMessage(PROJECT_NAME);
```

Comment:
Why do we need to wait for jdt.ls project import here?

Reply:
Rename operation is perform during the test. It fails when jdt.ls isn't started.